output_json/M1_flattened.json

```diff
@@ -0,0 +1,3373 @@
+[
```

Comment:
With iva-defects.json now in the resources directory, is this JSON still required? They look identical.